internal/webrisk_proto/webrisk.proto

```diff
@@ -100,25 +102,30 @@ message ComputeThreatListDiffResponse {
 // Request to check URI entries against threatLists.
 message SearchUrisRequest {
   // The URI to be checked for matches.
-  string uri = 1;
+  repeated string uris = 1;
```

Comment:
Won't it break backwards compatibility to change a type?

Reply:
Didn't think backwards compatibility would be an issue. We only use this is ipforensics, and wasn't planning to merge this back into upstream anyway. I don't think they really even maintain that library anymore.